packages/palette/src/elements/Image/Image.story.tsx

```diff
@@ -76,3 +76,20 @@ export const ImageLazyLoadSrcSet = () => {
 ImageLazyLoadSrcSet.story = {
   name: "Image + lazyLoad + srcSet",
 }
+
+export const ImageLazyLoadSrcSetBlurHash = () => {
```

Comment:
**suggestion:** Rename here and L93-94 to use `BlurDataURL` naming (since that is a meaningful difference vs `BlurHash`)